src/shared/openssl-util.c

```diff
@@ -1128,6 +1128,133 @@ int string_hashsum(
         return 0;
 }
 #  endif
+
+#if OPENSSL_VERSION_MAJOR >= 3
+DEFINE_TRIVIAL_CLEANUP_FUNC_FULL(OSSL_STORE_CTX*, OSSL_STORE_close, NULL);
+DEFINE_TRIVIAL_CLEANUP_FUNC_FULL(OSSL_STORE_INFO*, OSSL_STORE_INFO_free, NULL);
+#endif
```

Comment:
move to the openssl-util.h header, where the other cleanup funcs like this are placed?

Reply:
They are used only here, so no need to export them

Reply:
well, please keep them together with the others, otherwise people never look for this because it's hidden away here, and then 3 years later i have to do fixes such as 8fc798a9ad486d1810753f3249d136238d70a9fd.

This is a super-generic 1:1 wrapper around something that is public API, I think it should just be in openssl-util.h like all the other cleanup helpers we use only at once place.